include_core/j9nongenerated.h

```diff
@@ -135,6 +135,15 @@ typedef struct J9VMGC_SublistFragment {
 #define J9MMCONSTANT_EXPLICIT_GC_IDLE_GC 12
 #endif
 #define J9MMCONSTANT_IMPLICIT_GC_COMPLETE_CONCURRENT 13
+#define J9MMCONSTANT_EXPLICIT_GC_PREPARE_FOR_CHECKPOINT  14
+#if defined(OMR_GC_IDLE_HEAP_MANAGER)
+#define J9MM_IS_EXPLICIT_IDLE_GC(gcCode) ((gcCode) == J9MMCONSTANT_EXPLICIT_GC_IDLE_GC)
+#else
+#define J9MM_IS_EXPLICIT_IDLE_GC(gcCode) (1 == 0)
+#endif
+
+
+#define J9MM_IS_EXPLICIT_IDLE_OR_CHECKPOINT_GC(gcCode) (((gcCode) == J9MMCONSTANT_EXPLICIT_GC_PREPARE_FOR_CHECKPOINT) || J9MM_IS_EXPLICIT_IDLE_GC((gcCode)))
```

Comment:
I don't see much value in this OR macro. Having a macro for individual J9MM_IS_EXPLICIT_IDLE_GC and even for IS_CHECKPOINT_GC, if they are frequently used, is probably ok.